test/test_nn.py

```diff
@@ -15706,6 +15706,11 @@ def check(expected_out_shape, sizes, *args, **kwargs):
         y = torch.nn.functional.max_pool2d(x, 1, stride=(2, 2), padding=0, ceil_mode=True)
         self.assertEqual(y.size(), (1, 1, 3, 4))
 
+    def test_max_pool1d_neg_idx(self, device):
```

Comment:
Should this case be moved to OpInfo as error input?

Reply:
Thanks, Kshitij, I've added it :)

I'll add more test cases to ErrorInput in my next commit.